nav2_costmap_2d/plugins/static_layer.cpp

```diff
@@ -321,6 +324,9 @@ StaticLayer::updateCosts(
   if (!enabled_) {
     return;
   }
+  if (!map_received_) {
+    RCLCPP_ERROR(node_->get_logger(), "Can't update static costmap layer, no map received");
```

Comment:
This is going to get printed at 20Hz until the map message comes in. It shouldn't be an ERROR IMHO. Maybe just an INFO or WARn. But we should throttle it in some way so we're not getting so many messages

Reply:
I'd say `WARN` is good. +1 for throttling

Reply:
Changed to WARN with 1/10 throttling